modules/playing-card/assets/scss/frontend.scss

```diff
@@ -0,0 +1,32 @@
+.e-playing-cards-wrapper{
+	display: flex;
+	flex-wrap: wrap;
+	gap: 5px;
+
+	&-item{
+		display: flex;
+		flex-direction: column;
+		justify-content: space-between;
+		align-items: center;
+		width: 200px;
+		height: 400px;
+		box-shadow: 1px 3px 5px 0 rgba(184,184,184,0.75);
+		-webkit-box-shadow: 1px 3px 5px 0 rgba(184,184,184,0.75);
```

Comment:
I think these versions will be generated automatically.

Reply:
Correct, we have an auto pre-fixer.